### DISCUSSION TITLE
请教下redis使用集群的情况下的一些问题

### DISCUSSION THREAD

**Discussion by SheepQi** [cloudwu/skynet#1776] [Q&A]
我在看skynet/lualib/skynet/db/redis/cluster.lua的代码的时候发现nodes内有分别记录master和slave并且创建的时候也会有对应的read_slave的参数传入。但是我没太看明白在用cluster执行命令的时候是如何区分一个命令是只读的还是可写的。从而达成写命令走master node，读命令走slave的？

**Comment by ericzhanchina** [comment]
你如果想读写分离的话，我理解你需要在客户端和集群之间写一个Proxy才行吧
